d2core/d2render/ebiten/ebiten_panic_screen.go

```diff
@@ -0,0 +1,65 @@
+package ebiten
+
+import (
+	"github.com/hajimehoshi/ebiten/v2"
+	"github.com/hajimehoshi/ebiten/v2/ebitenutil"
+	"golang.org/x/image/colornames"
+)
+
+// PanicScreen represents the system to draw a panic message on the screen.
+type PanicScreen struct {
+	errorMessage string
+}
+
+// CreatePanicScreen creates a new panic screen.
+func CreatePanicScreen(errorMessage string) *PanicScreen {
+	result := &PanicScreen{
+		errorMessage: errorMessage,
+	}
+
+	ebiten.SetWindowResizable(true)
+
+	return result
+}
+
+// Update updates a game by one tick. The given argument represents a screen image.
+//
+// Update updates only the game logic and Draw draws the screen.
+//
+// In the first frame, it is ensured that Update is called at least once before Draw. You can use Update
+// to initialize the game state.
+//
+// After the first frame, Update might not be called or might be called once
+// or more for one frame. The frequency is determined by the current TPS (tick-per-second).
+func (s *PanicScreen) Update() error {
+	return nil
+}
```

Comment:
we may want to change the method comments to specifically describe the rationale for the `PanicScreen`